gapic/samplegen/samplegen.py

```diff
@@ -335,27 +335,24 @@ def validate_and_transform_request(self,
                 elif attr.enum:
                     # A little bit hacky, but 'values' is a list, and this is the easiest
```

Comment:
Do you still need this comment?

Reply:
I suppose it's not that helpful any more. I'll remove it in a subsequent PR.